service/history/workflowExecutionContext.go

```diff
@@ -217,6 +217,20 @@ func (c *workflowExecutionContext) updateHelper(builder *historyBuilder, transfe
 		replicationTasks = append(replicationTasks, c.msBuilder.createReplicationTask())
 	}
 
+	// // TODO
```

Comment:
remove this.

Reply:
If you want this for debugging then I would recommend to implement String() on mutableStateBuilder.

Reply:
good catch.